### PR TITLE
SDIT-1424 prison-api nomis-local spring profile is active in prod

### PR DESCRIPTION
**Pull request by steverendell**
Refactor config files

### CODE REVIEW

File: src/main/resources/application-nomis-hsqldb.yml [hunk 1]
```diff
@@ -15,9 +15,6 @@ spring:
       hikari:
         maximum-pool-size: 2
 
-  jpa:
-    show-sql: true
```
Comment (steverendell) [inline src/main/resources/application-nomis-hsqldb.yml:19]: Quieten verbose logging in tests by default

File: src/main/resources/application-test.yml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-spring:
-  security:
-    oauth2:
-      resourceserver:
-        jwt:
-          public-key-location: classpath:local-public-key.pub
```
Comment (steverendell) [inline src/main/resources/application-test.yml:6]: Not needed (file does not exist)

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -13,10 +13,8 @@ spring:
 
   profiles:
     group:
-      nomis-local:
-        - "connection-proxy"
       nomis:
-        - "nomis-local"
+        - "connection-proxy"
```
Comment (steverendell) [inline src/main/resources/application.yml:17]: Removed nomis-local profile and added its values to the nomis profile

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**